Sources/Foundation/Data.swift

```diff
@@ -2083,7 +2083,6 @@ public struct Data : ReferenceConvertible, Equatable, Hashable, RandomAccessColl
     }
 
     // slightly faster paths for common sequences
-    @inlinable // This is @inlinable as an important generic funnel point, despite being a non-trivial initializer.
```

Comment:
Why remove inlineability here?

Reply:
It was a cascading effect due to the use of `_withStackOrHeapBuffer` getting inlined, making this `inlinable` would expose the `@_implementationOnly` interface :-(.